nadlogar/problems/models/meta.py

```diff
@@ -54,14 +33,13 @@ class Problem(models.Model):
         on_delete=models.PROTECT,
         limit_choices_to=limit_content_type_choices,
     )
-    text = models.ForeignKey(
```

Comment:
Ali je mogoče, da z izbrisom tega kaj ne bi ustrezno delalo?

Reply:
Ideja te spremembe je, da ne :) Torej namesto `text`, ki je svoj model, imamo dve običajni polji `instruction` in `solution`. Treba je bilo veliko prilagotiev, ampak se mi zdi, da zdaj dela, kot bi moralo.